src/DssSpell.sol

```diff
@@ -38,13 +48,85 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
     // uint256 internal constant X_PCT_RATE      = ;
+    uint256 internal constant PT_TWO_FIVE_PCT_RATE = 1000000000079175551708715274;
 
     uint256 internal constant MILLION = 10 ** 6;
+    // uint256 internal constant RAY     = 10 ** 27;
     uint256 internal constant WAD     = 10 ** 18;
-    uint256 internal constant RAY     = 10 ** 27;
+
+    ChainLogLike internal immutable CHAINLOG    = ChainLogLike(DssExecLib.getChangelogAddress("CHANGELOG"));
+
+    address internal immutable FLASH_KILLER     = DssExecLib.getChangelogAddress("FLASH_KILLER");
+    address internal immutable MCD_FLASH        = DssExecLib.getChangelogAddress("MCD_FLASH");
+    address internal immutable MCD_FLASH_LEGACY = DssExecLib.getChangelogAddress("MCD_FLASH_LEGACY");
+
+    address internal immutable MCD_PSM_PAX_A    = DssExecLib.getChangelogAddress("MCD_PSM_PAX_A");
+    address internal immutable MCD_PSM_GUSD_A   = DssExecLib.getChangelogAddress("MCD_PSM_GUSD_A");
 
     function actions() public override {
 
+        // MOMC Parameter Changes
+        // https://vote.makerdao.com/polling/QmYUi9Tk
+
+        // Increase WSTETH-B Stability Fee to 0.25%
+        DssExecLib.setIlkStabilityFee("WSTETH-B", PT_TWO_FIVE_PCT_RATE, true);
+
+        // Increase Compound v2 D3M Maximum Debt Ceiling to 20 million
+        // Set Compound v2 D3M Target Available Debt to 5 million DAI (this might already be the case)
+        // TODO: Not on Goerli
+        // DssExecLib.setIlkAutoLineParameters("DIRECT-COMPV2-DAI", 20 * MILLION, 5 * MILLION, 12 hours);
+
+        // Increase the USDP PSM tin to 0.2%
+        DssExecLib.setValue(MCD_PSM_PAX_A, "tin", 2 * WAD / 1000);
```

Comment:
I know we go back and forth on the `20 * WAD / 10000);` vs `2 * WAD / 1000);` debate so I'll leave this up to you

Reply:
I'm not wedded to either way, I lifted this out of a previous spell. In any case we probably need to ensure that the PSM documentation is updated to clearly define this value.

Reply:
i kind of prefer not padding with 0s, my vote is leave as is

Reply:
The `20 * WAD / 10000` is more industry standard at this point.  I'm in favor of transitioning.